src/avfoundation.cs

```diff
@@ -7612,9 +7619,21 @@ interface AVCaptureVideoPreviewLayer {
 		[Static, Export ("layerWithSession:")]
 		AVCaptureVideoPreviewLayer FromSession (AVCaptureSession session);
 
+		[Static]
```

Comment:
Wrong pattern, i.e. that should not be static. I'm not sure if the registrars would run this correctly.